components/ledger/internal/services/command/send_overdraft_events.go

```
@@ -203,6 +203,7 @@ func buildOverdraftEvents(tran *transaction.Transaction) []overdraftEventItem {
 		return nil
 	}
 
+	//nolint:prealloc // contract: return nil (not empty slice) when no operation qualifies; tests assert nil.
```

golangci-lint (2.12.2)

[error] directive `//nolint:prealloc // contract: return nil (not empty slice) when no operation qualifies; tests assert nil.` is unused for linter "prealloc"

(nolintlint)